src/main/java/edu/wpi/first/smartdashboard/SmartDashboard.java

```diff
@@ -69,7 +69,8 @@ public void run() {
         1000);
 
     // Search the filesystem for extensions (49%)
-    FileSniffer.findExtensions(monitor, 0, 490);
+    //FileSniffer.findExtensions(monitor, 0, 490);
+    // Extensions disabled until classloading can be fixed
```

Comment:
You should probably add an issue and link to it from here.

Reply:
Done